reference/clinic.v1.yaml

```diff
@@ -184,6 +183,15 @@ paths:
         - CLINIC_ADMIN
       tags:
         - Clinics
+    delete:
```

Comment:
This is privileged, right? So should include:

```
x-roles:
        - TIDEPOOL_ADMIN
        - CLINIC_ADMIN
```

Reply:
Those headers don't do anything on the backend - we should get rid of them. But this comment reminded me that I didn't add the endpoint to the authorization policy. Nice catch!